Documentation/grpc-metadata.md

```diff
@@ -223,3 +223,54 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
     stream.SetTrailer(trailer)
 }
 ```
+
+## Updating metadata in the interceptor - server side
+
+Server side metadata updating in the interceptor examples are available [here](../examples/features/metadata_interceptor/server/main.go).
+
+#### Unary interceptor
+
+To read and update metadata in the interceptor under unary call,
```

Comment:
How about this?

```
The interceptor can read the existing metadata from the RPC context passed to it.
Since Go contexts are immutable, the interceptor will have to create a new context
with the updated metadata and pass it to the provided handler.
```

Reply:
Better than my previous expression. I will update it